src/library/baseplaylistfeature.cpp

```diff
@@ -478,6 +484,20 @@ void BasePlaylistFeature::slotExportPlaylist() {
     }
 }
 
+void BasePlaylistFeature::slotExportTrackFiles() {
+    int rows = m_pPlaylistTableModel->rowCount();
+    QList<TrackPointer> tracks;
+    for (int i = 0; i < rows; ++i) {
+        QModelIndex index = m_pPlaylistTableModel->index(i, 0);
+        tracks.push_back(m_pPlaylistTableModel->getTrack(index));
+    }
+
+    DlgTrackExport track_export_dlg(nullptr, m_pConfig, tracks);
```

Comment:
I think I can do a little refactoring in a way that doesn't confuse things too much